src/components/pages/home/HomePage.js

```diff
@@ -1,10 +1,16 @@
 import React, { Component } from 'react';
-import TempPage from '../temp';
+import PageHeading from '../../shared/PageHeading';
```

Comment:
While we're here, let's add a `PageHeading` component to the `PhotoPage` and `WeatherPage`